DiffusionPreprocessing/DiffPreprocPipeline_Eddy.sh

```diff
@@ -223,7 +223,7 @@ get_options() {
 	resamp_value=""
 	unset ol_nstd_value
 	extra_eddy_args=""
-	no_gpu=""
+	no_gpu="False"
```

Comment:
Our changing convention on the case across these scripts is annoying (False vs false vs FALSE), but I see that what you did in `DiffPreprocPipeline_Eddy.sh` is consistent with the rest of the script.